efficientnet/scripts/convert_efficientnet.sh

```diff
@@ -0,0 +1,216 @@
+  # Checks whether $1 is in the array $2
+  # Example:
+  # The snippet
+  #   array=("1" "a string" "3")
+  #   containsElement "a string" "${array[@]}"
+  # Returns 1
+  # Source:
+  # https://stackoverflow.com/questions/3685970/check-if-a-bash-array-contains-a-value
+  local e match="$1"
+  shift
+  for e; do [[ "$e" == "$match" ]] && return 0; done
+  return 1
+}
+
+# The mechanics of reading the long options are taken from
+# https://mywiki.wooledge.org/ComplexOptionParsing#CA-78a4030cda5dc5c45377a4d98ebd4fe610e0aa7e_2
+usage() {
+  echo "Usage:"
+  echo "  $0 [ --help | -h ]"
+  echo "  $0 [ --target_dir=<value> | --target_dir <value> ] [options]"
+  echo
+  echo "Options:"
+  echo "  --use_venv=true|yes|1|t|y :: Use the virtual env with pre-installed dependencies"
+  echo
+  echo "The default target_dir is dist."
+}
+
+# set defaults
+LAST_ARG_IDX=$(($# + 1))
+TARGET_DIR=$(realpath "dist")
+USE_VENV="false"
+declare -A LONGOPTS
+# Use associative array to declare how many arguments a long option
+# expects. In this case we declare that loglevel expects/has one
+# argument and range has two. Long options that aren't listed in this
+# way will have zero arguments by default.
+LONGOPTS=([target_dir]=1 [use_venv]=1)
+OPTSPEC="h-:"
+while getopts "$OPTSPEC" opt; do
+  while true; do
+    case "${opt}" in
+    -) #OPTARG is name-of-long-option or name-of-long-option=value
+      if [[ ${OPTARG} =~ .*=.* ]]; then # with this --key=value format only one argument is possible
+        opt=${OPTARG/=*/}
+        ((${#opt} <= 1)) && {
+          echo "Syntax error: Invalid long option '$opt'" >&2
+          exit 2
+        }
+        if (($((LONGOPTS[$opt])) != 1)); then
+          echo "Syntax error: Option '$opt' does not support this syntax." >&2
+          exit 2
+        fi
+        OPTARG=${OPTARG#*=}
+      else #with this --key value1 value2 format multiple arguments are possible
+        opt="$OPTARG"
+        ((${#opt} <= 1)) && {
+          echo "Syntax error: Invalid long option '$opt'" >&2
+          exit 2
+        }
+        OPTARG=(${@:OPTIND:$((LONGOPTS[$opt]))})
+        ((OPTIND += LONGOPTS[$opt]))
+        echo $OPTIND
+        ((OPTIND > $LAST_ARG_IDX)) && {
+          echo "Syntax error: Not all required arguments for option '$opt' are given." >&2
+          exit 3
+        }
+      fi
+      continue
+      ;;
+    target_dir)
+      TARGET_DIR=$(realpath $OPTARG)
+      ;;
+    use_venv)
+      USE_VENV=$OPTARG
+      ;;
+    h | help)
+      usage
+      exit 0
+      ;;
+    ?)
+      echo "Syntax error: Unknown short option '$OPTARG'" >&2
+      exit 2
+      ;;
+    *)
+      echo "Syntax error: Unknown long option '$opt'" >&2
+      exit 2
+      ;;
+    esac
+    break
+  done
+done
+
+# internal variables
+TRUE=(
+  "true"
+  "t"
+  "1"
+  "yes"
+  "y"
+)
+CONVERTED_MODELS_DIR="efficientnet"
+VIRTUALENV_DIR="venv"
+MODEL_SOURCE_DIR="efficientnet-source"
+MODEL_SOURCE_URL="https://github.com/sdll/efficientnet"
```

Comment:
We should source the checkpoint from an official tfjs bucket and remove dependence on the forked repository